tools/depends/target/exiv2/Makefile

```diff
@@ -0,0 +1,40 @@
+include ../../Makefile.include EXIV2-VERSION ../../download-files.include
+DEPS = ../../Makefile.include Makefile EXIV2-VERSION ../../download-files.include 0001-Remove-filesystem-dependency.patch
+
+# configuration settings
+CMAKE_OPTIONS=-DCMAKE_INSTALL_PREFIX=$(PREFIX) \
+			  -DEXIV2_ENABLE_WEBREADY=OFF \
```

Comment:
Indentation here is a mix of tabs and spaces, maybe remove the 3 tabs and just use 2 spaces?